lib/Drush/Boot/DrupalBoot8.php

```diff
@@ -43,7 +48,9 @@ function bootstrap_drupal_database() {
   }
 
   function bootstrap_drupal_configuration() {
-    $this->drupal8_bootstrap(DRUSH_BOOTSTRAP_DRUPAL_CONFIGURATION);
+    $request = Request::createFromGlobals();
+    $classloader = drush_drupal_load_autoloader(DRUPAL_ROOT);
```

Comment:
This is just the existing Drush bootstrap code for Drupal 8, refactored--unless I made a mistake.  :)   In any event, if there is anything that could be improved or made more efficient here, we'd be glad to make adjustments.